go_client/imagenetTF.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import "github.com/go-redis/redis"
+
+import (
+    "fmt"
+    "image"
+    "image/jpeg"
+    "image/draw"
+    "os"
+    "bytes"
+    "encoding/binary"
+    "io/ioutil"
+    "encoding/json"
+    "strconv"
+)
+
+func init() {
+    // Can only read jpeg
+    image.RegisterFormat("jpeg", "jpeg", jpeg.Decode, jpeg.DecodeConfig)
+}
+
+
+func getRGBImage(imgPath string) (*bytes.Buffer, image.Rectangle) {
+    // It is not the optimized read
+    var rgbImg []uint8
+    imgfile, _ := os.Open(imgPath)
+    defer imgfile.Close()
+    decodedImg, _, _ := image.Decode(imgfile)
+    rect := decodedImg.Bounds()
+    rgba := image.NewRGBA(rect)
+    draw.Draw(rgba, rect, decodedImg, rect.Min, draw.Src)
+    for x:= 0; x < len(rgba.Pix); x += 4 {
+        rgbImg = append(rgbImg, rgba.Pix[x], rgba.Pix[x + 1], rgba.Pix[x + 2])
+    }
+    imgbuf := new(bytes.Buffer)
+    binary.Write(imgbuf, binary.BigEndian, rgbImg)
+    return imgbuf, rect
+}
```

Comment:
So the issue here is that we need to discard the alpha channel, right? We could actually do this more efficiently with a SCRIPT.
Or at least we should pre-allocate `rgbImg` to the right size, `append` doesn't look right.

Reply:
I agree and I wasn't happy with that either. Wanted to make it comparable with JS and Python version. So keeping it to SCRIPT is not a choice (I'll probably comment it there so that readers would know). I should pre-allocate.